src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1934,6 +1936,7 @@ void PgScrubber::dump_scrubber(ceph::Formatter* f,
       m_scrub_job->scheduling_state(ceph_clock_now(), deep_expected);
     f->dump_string("schedule", sched_state);
   }
+  f->dump_int("test_sequence", m_sessions_counter);  // an ever-increasing number used by tests
```

Comment:
Adding this here will make it part of the user-facing "pq query" output. Is there any reason why we should expose this to users? `test_` does not make sense in any output that users will consume.

Reply:
Hmmm... I was under the impression that the 'query' command is not for 'regular users'.
Marking the PR as DNM for now.

@neha-ojha: I see two options:
- 1) keep the field, renaming it to 'scrub_sessions' (although it doubly-count each session), or/and
- 2) make the field dependent on an activation command via the asok

I don't want to give up on this field altogether - it is much too handy in stabilizing and shortening tests.